services/essentials.yaml

```diff
@@ -78,27 +78,45 @@ spec:
     - groupId: org.jenkins-ci.plugins.workflow
       artifactId: workflow-aggregator
       version: '2.6'
+    - groupId: org.jenkins-ci.plugins
+      artifactId: durable-task
+      version: '1.28'
     - groupId: org.jenkins-ci.plugins.workflow
       artifactId: workflow-api
       version: '2.33'
+    - groupId: org.jenkins-ci.plugins.workflow
+      artifactId: workflow-basic-steps
+      version: '2.13'
     - groupId: org.jenkins-ci.plugins.workflow
       artifactId: workflow-cps
-      version: '2.60'
+      version: '2.61'
     - groupId: org.jenkins-ci.plugins.workflow
       artifactId: workflow-cps-global-lib
       version: '2.12'
+    - groupId: org.jenkins-ci.plugins.workflow
+      artifactId: workflow-durable-task-step
+      version: '2.27'
     - groupId: org.jenkins-ci.plugins.workflow
       artifactId: workflow-job
-      version: '2.29'
+      version: '2.31'
     - groupId: org.jenkins-ci.plugins.workflow
       artifactId: workflow-scm-step
       version: '2.7'
+    - groupId: org.jenkins-ci.plugins.workflow
+      artifactId: workflow-step-api
+      version: '2.17'
     - groupId: org.jenkins-ci.plugins.workflow
       artifactId: workflow-support
-      version: '2.22'
+      version: '3.0'
```

Comment:
That's fine. That's a great fix we were waiting on for the Java 11 support, and in the end is also compatible with Java 8.
And we don't expect issues with it.